src/components/UrlBlacklist/index.jsx

```diff
@@ -0,0 +1,288 @@
+  // const dispatch = useDispatch();
+  const [pageCount] = useState([1, 2, 3, 4]);
+
+  useEffect(() => {
+    const list = [
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh1',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh2',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh3',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh4',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh5',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh6',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh7',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh8',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh9',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdhA',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdhB',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdhC',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdhD',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdhE',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdhF',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+    ];
+    setListuser(list);
+  }, []);
+
+  const handleAdding = () => {
+    try {
+      if (addlink.length <= 0) throw Error('Please Enter the link');
+      listuser.forEach((el) => {
+        if (el.link === addlink) throw Error(`This link has been added at ${el.addedat}`);
+      });
+      const current = new Date();
+      const clearFormat = (num) => (num > 9 ? `${num}` : `0${num}`);
```

Comment:
hàm này nên đặt là vầy nha em
```suggestion
      const formatNumber = (num) => (num > 9 ? `${num}` : `0${num}`);
```